src/classes/ADDR_Contact_TDTM.cls

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
* This variable doesn't seem to have a very meaningful name. I suggest to give this variable a concise but meaningful name that suggest its purpose.
* Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.